tsconfig.json

```diff
@@ -15,7 +17,6 @@
     "dist",
   ],
   "typeRoots": [
-    "node_modules/@types",
-    "node_modules/web3-typescript-typings"
```

Comment:
this is no longer needed? I think it might be, when this package is used by another package

Reply:
Vanille doesn't seem to need it.

Reply:
BTW, I did a thorough check this time that arc.d.ts compiles.

Reply:
(so does the javascript!)